native_words.asm

```diff
@@ -644,7 +644,7 @@ xt_again:
                 ; so we have the range and don't have to calculate the
                 ; offset.
                 ldy #0
-                lda #$4C        ; JMP
+                lda #OpJMP
```

Comment:
This is excellent and makes things much clearer for those that don't know the opcode values.

Reply:
i'll fix a couple of these in taliforth.asm also